docs/SOPs/data_retraction_SOP.md

```diff
@@ -103,19 +140,18 @@ rm -r /nfs/production/hca/<name_of_the_folder>/
 **Tips to find the folder**: It is usually best practice to set up the name with the shortname of the project, if you know this dataset was archived but there is no apparent folder, please contact the primary wrangler for this project. The primary wrangler's name can be found in the GitHub ticket.
 
 ### Staging area
+1. Find the directory of the project in the Terra staging area GCP bucket. The GCP buckets locations are configured in `<env>.yaml` files in [ingest-kube-deployment/apps](https://github.com/ebi-ait/ingest-kube-deployment/tree/master/apps)
 
-1. Run the script [here](https://github.com/ebi-ait/hca-ebi-dev-team/tree/master/scripts/map_ingest_uuid_to_staging_area) to get a mapping between the uuid of the entities in ingest and the file path in the staging area.
-1. Use that file as the input for the following commands:
+    ```
+    # Example project directory in dev:
+   
+    gs://broad-dsp-monster-hca-dev-ebi-staging/dev/<project-uuid> 
+    ```
+  
+2. Remove the project uuid subdirectory:
    ```
-   cat <name_of_mapping_file> |  jq -r ' .[] | .[]' > file_paths.txt
-   cat file_paths.txt | gsutil -m rm -I
+   gsutil -m rm gs://<bucket-parent-directory>/<project-uuid>
```

Comment:
@ESapenaVentura it should be easier to delete files for a project now. Does these instructions look good to you?